README.md

```diff
@@ -28,11 +28,8 @@ Installing HDFS-Mesos on your Cluster
 3. Optional: Customize any additional configurations that weren't updated at compile time in `hdfs-mesos-*/etc/hadoop/*-site.xml` Note that if you update hdfs-site.xml, it will be used by the scheduler and bundled with the executors. However, core-site.xml and mesos-site.xml will be used by the scheduler only.
 4. Check that `hostname` on that node resolves to a non-localhost IP; update /etc/hosts if necessary.
 
-### If you have Hadoop pre-installed in your cluster
-If you have Hadoop installed across your cluster, you don't need the Mesos scheduler application to distribute the binaries. You can set the `mesos.hdfs.native-hadoop-binaries` configuration parameter in `mesos-site.xml` if don't want the binaries distributed.
 
```

Comment:
that was text previous to me... and this review is on text that was removed... or pasted in another place... 
I'll fix it there